fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonDeviceHL.xtend

```diff
@@ -1,386 +1,388 @@
-//+======================================================================
```

Comment:
Don't know why the complete file is like this...

Reply:
I am also not sure. I have done git rebase with Upstream master before pushing new commits.

Reply:
My guess would be that the line breaks have changed.  Maybe from `\r\n` to `\n`, or vice versa.